static_src/components/header.jsx

```diff
@@ -0,0 +1,60 @@
+
+import React from 'react';
+
+import createStyler from '../util/create_styler';
+
+import headerStyle from 'cloudgov-style/css/components/header.css';
+import navStyle from 'cloudgov-style/css/components/nav.css';
+import logoStyle from 'cloudgov-style/css/components/logo.css';
+
+export default class Header extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.styler = createStyler(headerStyle, navStyle, logoStyle);
+  }
+
+  getImagePath(iconName) {
```

Comment:
Yea I can't think of a good place for it, never mind. I just meant that it seems like a thing many components will use.